docs/Documentation.md

```diff
@@ -286,6 +286,65 @@ That said, most of the preferences are very simple.
 But they demonstrate the flexibility of Fantomas on a set of configurations. 
 More preferences will be added depending on use cases.
 
+##### "AlignBrackets"
+
+Alternative style to format records, arrays and lists.
+If enabled, this will align the brackets (`{`, `{|`, `[`, `[|`) and (`}`, `|}`, `]`, `|]`) on the same line.
```

Comment:
It seems it should've been `on separate lines` here?

Reply:
Separate lines isn't always the case. For example: https://github.com/fsprojects/fantomas/pull/706/files#diff-2bb5a182cc5d09abf8ec12ca99f7d645R93

That is why I felt like align brackets was a description that covered all scenarios.
`BracketsOnSameColumn` might be an alternative, I'm definitely open to suggestions.

Reply:
I see, thanks. The on the same line part looks misleading, though. I'm trying to come up with a better name.

Reply:
Maybe something like `MultilineBlockBracketsOnSeparateLines`?

Reply:
Maybe `MultilineBlockBracketsOnSameColumn` ?
The `OnSeparateLines` part might create confusing for
```fsharp
{ new System.Object() with
            member x.ToString() = "F#"
}
```
You could get the impression that `{` should be on a separate line.
However, this is by design not the case for object expressions and record updates.